website/docs/components/form/sensitive-input/partials/accessibility/accessibility.md

```diff
@@ -0,0 +1,49 @@
+
+## Keyboard navigation
+
+<Doc::Badge @type="neutral">Tab</Doc::Badge>
+
+Focus on the input
+
+![Keyboard sensitive input focus example](/assets/components/form/sensitive-input/sensitive-input-focus.png =836x*)
+
+<Doc::Badge @type="neutral">Tab</Doc::Badge>
+
+Move focus to the toggle button
+
+![Keyboard sensitive input toggle focus example](/assets/components/form/sensitive-input/sensitive-input-toggle-focus.png =836x*)
+
+<Doc::Badge @type="neutral">Spacebar</Doc::Badge>
+<Doc::Badge @type="neutral">Enter</Doc::Badge>
+
+Activates toggle to show/hide the input value
+
+![Keyboard sensitive input toggle focus example](/assets/components/form/sensitive-input/sensitive-input-toggle-activated.png =836x*)
```

Comment:
@cveigt instead of "Filled" here I would put some text that resembles a code/ID (with the same number of character as the dots above)

Reply:
Updated. Thanks for the suggestion!